content/post/a-place-to-start-gdd.md

```diff
@@ -0,0 +1,69 @@
+
+**That's what we wanted: a description of the game, made of some sketches, graphs, lines of dialogue.**
+
+Second: we thought that working with a GDD as our expected outcome could help us doing this side project one step at a time. The GDD would act as a sort of guideline, forcing a sustainable pace.
+You probably know this: work, family, and life in general, leaves you with little time for your stuff, so we found it was important to focus on a process that could give us small steps and frequent rewards.
+
+**If you are an 10pm warrior like us, don't underestimate the need to get rewarded and take fun off your project, or you will risk to eventually lose interest in it.**
+
+Third: The idea of a GDD helps us a lot to focus on the value of what we are doing, because beside the work for yourself, it forces you to answer a perpetual question: is the structure of what I'm writing here clear enough for a reader? Am I giving something for granted?
+We gave ourselves a rough list of steps to start with, some points that we thought would be essential to understand the big picure of the game, using this question as a guideline. So far is working quite well.
+
+**Walking in your target's shoes helps providing valuable information and valuable information is what you need to craft a great product.**
+
+Fourth: a GDD it is a good starting point for the future. Let's say you decide to take your project to the next step, and find a company to develop it; the GDD - even though it is clearly not the tool to _sell_ your idea - is, in my opinion, a great tool to describe your game and it highlights the fact you have actually designed it, and took it seriously as a project.
+
+**A good design talks about your love for your project. A GDD can shout it out loud.**
+
+
+## What we are writing in our GDD
+
+When we started writing the document we also tried to create a list of what we thought it should have contained. A list helps us to undestand how much work still has to be done and, while working on a step, to note down anything relevant that could be useful in the next ones.
+
+This is the table of contents we made.
+Right now some of these points are far from done (not even started, actually) but that's part of the plan: we write down what we think it will be useful and, as soon as we get there, we decide if it still is. We'll try to update you about this in some future posts.
+
+* a seed, the abstract of the game plot
+* a brief description of the plot itself divided in four acts
+* a small section of aestethic influences
+ * what styles of graphics we like for a GA
+ * what styles we think are suitable for our GA
```

Comment:
What's the difference between this line and the preceding one?
I'd restate this in another form since I didn't get it...